extensions-contrib/kafka-emitter/src/main/java/org/apache/druid/emitter/kafka/KafkaEmitterConfig.java

```diff
@@ -23,22 +23,49 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.base.Preconditions;
 import com.google.common.collect.ImmutableMap;
+import org.apache.commons.lang3.EnumUtils;
 import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.errors.InvalidConfigurationException;
 
 import javax.annotation.Nullable;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
 import java.util.Map;
+import java.util.Set;
 
 public class KafkaEmitterConfig
 {
+  public enum EventType
+  {
+    metrics,
+    alerts,
+    requests,
+    segment_metadata
```

Comment:
by convention enum constants are always uppercase in Java. I understand you did this to allow de-serializing lower-case values, but that can be solved by providing a @JsonCreator annotated method like I had linked to in my previous comment about using enums.